### ISSUE TITLE
Facing problem with multiple gesture detection

### ISSUE THREAD

**Issue by ceosss** [flutter/flutter#85455]
Hey,
I'm facing a problem with multiple gesture detection in flutter.

I have a unity widget attached to flutter ([Using this library](https://pub.dev/packages/flutter_unity)).
Unity widget provides a game-like experience,
you can use the joystick on the left to move and the empty area on the right to look around.

But somehow flutter only allows both the controls to work together only once, for the first time.
eg, if I use both the controls together for the first time, it will work fine I can move and look around at the same time.
Now  if I lift a finger and then put it back and try to move and look around at the same time, both controls don't work together, 
I've to lift both my fingers and try again to make both of them work together.

My Questions

1. Is this expected behaviour?
2. Is this because of GestureArena, if yes, is there any way I can disable it?
3. Is there a workaround to solve this?

Thanks!

**Comment by TahaTesser** [comment]
Hi @ceosss 
Can you please provide your `flutter doctor -v`, a complete reproducible minimal code sample
Thank you

**Comment by TahaTesser** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
